bin/post-rewrite.js

```diff
@@ -13,9 +13,9 @@ const rl = createInterface({
   input: process.stdin,
 });
 
-rl.on('line', line => {
+rl.on('line', (line) => {
```

Comment:
I can't imagine the flame wars that lead to this change.

Reply:
This is a configuration option, which means a lot for prettier :D
I kept the default because their ratonale makes sense (it's easier to add a type or add more parameters when the parentheses are already there).

More controversial (to me at least) is the decision to transform `function()` into `function ()` :D